scripts/animatediff_ui.py

```diff
@@ -88,8 +88,8 @@ def set_p(self, p):
             self.video_default = False
         if self.overlap == -1:
             self.overlap = self.batch_size // 4
-        if "PNG" not in self.format:
-            p.do_not_save_samples = True
+        # _save will save frames to subfolders when saving PNG output
+        p.do_not_save_samples = True
```

Comment:
pls make this feature optional, by having an optional setting in `Settings/AnimateDiff`

Reply:
Good idea, I named the option after p.do_not_save_samples. Off by default.